tests/hf/test_integrals.py

```diff
@@ -274,3 +277,156 @@ def test_gradient(symbols, geometry, alpha, coeff):
 
     assert np.allclose(g_alpha, g_ref_alpha)
     assert np.allclose(g_coeff, g_ref_coeff)
+
+
+@pytest.mark.parametrize(
+    ("i", "j", "ri", "rj", "alpha", "beta", "d"),
+    [
+        # _diff2 must return 0.0 for two Gaussians centered far apart at 0.0 and 20.0
```

Comment:
Do we have a number to compare with for the case of non-zero overlap? If yes, I would suggest to add it.